pkg/server/router/common.go

```diff
@@ -31,25 +31,40 @@ import (
 // to Router and Gateway.
 type commonRouter struct {
 	payloadFunc      func(req *http.Request) (p *Payload, err error)
-	matchHandlerFunc func(req *http.Request, p *Payload) (h Handler, pp []Processor)
+	matchHandlerFunc func(path string, method string, p *Payload) (h Handler, pp []Processor)
```

Comment:
After reading it with the `router.Payload`. I think the matchHandlerFunc should only accept `Payload` alone. The action string should be a part of data. The HTTP method should be part of Meta?

I think `Meta` is designed for that purpose, but implementation just ignore it.